lib/Review_Ride.dart

```diff
@@ -84,44 +92,46 @@ class _ReviewRideState extends State<ReviewRide> {
             ),
             SizedBox(height: 30),
             Row(
-              mainAxisAlignment: MainAxisAlignment.start,
-              children: <Widget>[
-                Text('From', style: labelStyle)
-              ]
-            ),
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: <Widget>[Text('From', style: labelStyle)]),
             SizedBox(height: 5),
             Row(
               mainAxisAlignment: MainAxisAlignment.start,
               children: <Widget>[
-                Text('Bill & Melinda Gates Hall', style: infoStyle)
+                Text(
```

Comment:
Instead of checking that the fields aren't null, we should probably guarantee that location isn't null in the flow, right? (applies to all fields in the ride below as well) Again, we can probably save validation for a later PR